RELEASE.md

```diff
@@ -0,0 +1,107 @@
+# Release Guide
+
+This document introduces how to publish a new release of Chaos Mesh.
+
+## How to publish a new release
+
+Here are several steps to publish a new release, and several of them are optional for bugfix/patch release.
+
+### Step 1 Draft Release Notes
+
+This step is required for all the releases(major, minor, bugfix/patch).
+
+As we maintain a CHANGELOG.md on each active branch, we could use it to draft release notes. Changelog is used for developers and Release Notes is used for users, so there are some little different between them.
+
+- Draft a Release Notes on google docs, here is a Release Notes template: [Google Docs](https://docs.google.com/document/d/1v0P5NQyepEyT4CH8usouyJup_fvOYtsYAz8nbJfn3Jk/edit?usp=sharing)
+- Collect changelogs from `CHANGELOG.md` on the target branch into Release Notes. There are several patterns for mapping them:
+  - Section `Add` in `CHANGELOG.md` is mapped to `New Features` in Release Notes.
+  - Section `Changed` with actual "enhanced changes" in `CHANGELOG.md` is mapped to `Enhancements` in Release Notes.
+  - Section `Fixed` in `CHANGELOG.md` is mapped to `Bug Fixes` in Release Notes.
+  - For the rest of `CHANGELOG.md`, put them into `Others` in Release Notes.
+- After finishing the draft, send it to [Chaos Mesh Committers and Maintainers](https://github.com/chaos-mesh/chaos-mesh/blob/master/MAINTAINERS.md), ask them to review it.
+- Once 2 (or more) of committer/maintainers left "Approve" on the draft Release Note, it's ready for making a release on GitHub.
+
+### Step 2 Checkout New Branch
+
+This step is only required for major or minor version release. You should skip this step if you are going to release a bugfix/patch version.
+
+Checkout a new branch with the name of `release-x.y` from `master` where `x.y` is the semantic version number of the release.
+
+### Step 3 Update Versions in Helm Charts and install.sh
+
+This step is required for all the releases(major, minor, bugfix/patch).
+
+You should make a new PR for updating the version in helm charts and `install.sh`:
+
+- `version` and `appVersion` in `helm/chaos-mesh/Chart.yaml`. Please notice that do NOT use prefix `v` in here.
+- Docker image tags in helm charts:
+  - After `2.1.0`, there is only one place need to change: `images.tag` in `helm/chaos-mesh/values.yaml`.
+  - Before `2.1.0`, you should update the image tag for **each image**
+- `version` in help messages of `install.sh`, about line 46.
```

Comment:
> about line 46

After https://github.com/chaos-mesh/chaos-mesh/pull/2797, it should be line 45 now 😢  maybe it's not a good idea to clarify the line number.

Reply:
I change the way to describe it , PTAL :D